easybuild/easyconfigs/c/CP2K/CP2K-6.1-fosscuda-2019a.eb

```diff
@@ -0,0 +1,36 @@
+name = 'CP2K'
+version = '6.1'
+
+homepage = 'http://www.cp2k.org/'
+description = """CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
+ simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
+ methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
+ classical pair and many-body potentials. """
+
+toolchain = {'name': 'fosscuda', 'version': '2019a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/cp2k/cp2k/releases/download/v%(version)s.0/']
+sources = [SOURCELOWER_TAR_BZ2]
+patches = ['CP2K-2.4.0-fix_compile_date_lastsvn.patch']
+checksums = [
+    'af803558e0a6b9e9d9ce8a3ab955ba32bacd179922455424e061c82c9fefa34b',  # cp2k-6.1.tar.bz2
+    '02475cbe24c8d4ba27037c826adf8a534cad634c3c4e02c21d743f5284516bda',  # CP2K-2.4.0-fix_compile_date_lastsvn.patch
+]
+
+builddependencies = [
+    ('flex', '2.6.4'),
+    ('Bison', '3.0.5'),
+]
+
+dependencies = [
+    ('libxsmm', '1.10'),
+    ('CUDA', '10.1.105'),
+]
```

Comment:
@Micket The non-CUDA easyconfig for `CP2K` also has the following dependencies in addition to `libxsmm`:
```
('Libint', '1.1.6'),
('libxc', '4.3.4'),
('PLUMED', '2.5.1'),
```
Does the CUDA version not need such dependencies?
Also, `('CUDA', '10.1.105')` should not be necessary since it is included in `fosscuda`.

Reply:
Even when tried building it with these included, the test-suite skipped all test involving them.
This gave me the impression that they weren't support in the GPU-build.

I could definitely be wrong here.